testpilot/metrics/views.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.conf import settings
+from django.http import Http404
+
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.response import Response
+from rest_framework.decorators import permission_classes, api_view
+
+DEFAULT_LOG_PING_WHITELIST = [
+    'testpilottest'
+]
+
+
+@csrf_exempt
+@api_view(['POST'])
+@permission_classes([])
+def log_ping(request, logger_name):
+    """Accept and log metrics pings"""
+    whitelist = getattr(settings, 'LOG_PING_WHITELIST', DEFAULT_LOG_PING_WHITELIST)
```

Comment:
No, this should be like [the general `testpilottest` ping in Telemetry](https://github.com/mozilla/testpilot/blob/master/docs/README-METRICS.md#data-collection-details). One log event type (and URL) for all experiments. You should include the experiment's add-on ID in the JSON payload. Don't really have a schema for that payload, but i figured it could look like the telemetry ping.